graviti/schema/catalog_to_schema.py

```diff
@@ -0,0 +1,271 @@
+#!/usr/bin/env python3
+#
+# Copyright 2022 Graviti. Licensed under MIT License.
+#
+
+"""Functions converting TensorBay Catalog to Graviti Schema."""
+
+from functools import reduce
+from pathlib import Path
+from typing import Any, Callable, Dict, List, Union
+
+from tensorbay.dataset import Notes, RemoteData
+from tensorbay.label import (
+    Catalog,
+    ClassificationSubcatalog,
+    InstanceMaskSubcatalog,
+    Keypoints2DSubcatalog,
+    Label,
+    LabeledKeypoints2D,
+    PanopticMaskSubcatalog,
+)
+from tensorbay.label.basic import SubcatalogBase, _LabelBase
+from tensorbay.label.supports import AttributesMixin, CategoriesMixin, MaskCategoriesMixin
+
+_SUFFIX_AND_FILE_TYPES = {
```

Comment:
the name of this dict should be updated.
also I think you can generate this dict by a function rather than just write it down.
because there are duplicate items in it.